invictum-test-core/src/main/java/com/github/invictum/panels/AbstractPanel.java

```diff
@@ -77,15 +79,18 @@ protected String locatorValue(final String locatorKey) {
     }
 
     protected By locator(final String locatorKey, final String... parameters) {
+        parentPage.addJQuerySupport();
```

Comment:
As for me it's bad practice to add JQuerry for all panel locators, even if user didn't use JQuerry at all.